github_issue_tools/README.md

```diff
@@ -17,10 +17,6 @@ GitHub issue tools provide a bash script to help automatically create GtiHub iss
     - Configure the `.env` file created before. Enter your `github personal access token` in the `token` field and `issue.csv` file location in the `csv_file` field.
 
 ## How to use
-- Provide the executable permissions to the script by the command:
-```
-chmod +x create_github_issue.sh
-```
 - Run the script to create GitHub issues in the repository:
 ```
 ./create_github_issue.sh
```

Comment:
At tis current state, this script relies on `jq` being installed, so we should mention that it's required here